pyaerocom/aeroval/setupclasses.py

```diff
@@ -396,16 +496,19 @@ def to_json(self, outdir: str, ignore_nan: bool = True, indent: int = 3) -> None
         return filepath
 
     @staticmethod
-    def from_json(filepath: str) -> "EvalSetup":
+    def from_json(filepath: str) -> EvalSetup:
```

Comment:
maybe annotate with return `Self` from [PEP 673]?

[PEP 673]: https://peps.python.org/pep-0673/

Reply:
Sure, good suggestion. `EvalSetup` doesn't currently have any child classes but good idea to get ahead of any issue.